packages/app-desktop/app.ts

```diff
@@ -707,6 +707,10 @@ class Application extends BaseApplication {
 
 		// await runIntegrationTests();
 
+		if (Setting.value('env') === 'dev') {
+			await runCryptoIntegrationTests();
+		}
```

Comment:
That's not needed on desktop, because any test can already run using Jest. So please move any such test to Jest.

Reply:
This has been added to `crypto.test.ts` so I just removed this test.

Reply:
I just found the test doesn't work well in Jest. I have fixed it in commit 95281e7460fbda1fb083443c1aa390bc9fc41d6e